### PR TITLE
Add Blink functionality in Atmega32

### LINKED ISSUE #271: Add Blink simulation and code for Atmega32
About: Add LED Blink code for Atemga32
Label: enhancement

Define You:
- [x] SSOC Participant
- [ ] Contributor
- [ ]  Mentor

Describe the Feature:
Adding a simple LED blink code feature in Atmega32A (latest version of atmega32) Microcontroller.

### PR DESCRIPTION
**Pull request by khanasif786**
Fixes #271

### CODE REVIEW

**Comment by khanasif786** [comment]
@Harshil-Jani I am currently adding simulation and video files in separate commit on same branch.

**Comment by Harshil-Jani** [comment]
@khanasif786 Hey ! Can you look at this regulations https://github.com/prathimacode-hub/IoT-Spot/blob/main/CONTRIBUTING.md#other-points-to-remember-while-submitting-your-work
And Please make the required changes so that we can get your work merged.

**Review by Harshil-Jani — CHANGES_REQUESTED** [review]
https://github.com/prathimacode-hub/IoT-Spot/blob/main/CONTRIBUTING.md#other-points-to-remember-while-submitting-your-work
Apply changes as mentioned here.

**Comment by khanasif786** [comment]
@Harshil-Jani . I haven't made the final work yet. But could you please give me specific feedback regarding this.

**Comment by Harshil-Jani** [comment]
@khanasif786 Maybe if the work is still in progress then We can convert this to draft PR. 
So far the feedback is concerned your code looks good. But make sure it follows the guidelines and the filenames and structure of directories is upto mark.

**Comment by Harshil-Jani** [comment]
And let me know when you are done with the work ! We can then bring it out of the Draft PR.

File: Atmel/LED Blinking/led_blinking.c [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+{
+	DDRB = 0xff;
+	while (1)
+	{
+		PORTB = 0xff;
+		_delay_ms(1000);
+		PORTB = 0x00;
+		_delay_ms(1000);
+	}
+}
```
Comment (Harshil-Jani) [inline Atmel/LED Blinking/led_blinking.c:15]: Add New Line at End of File (EOF)

**Review by Harshil-Jani — CHANGES_REQUESTED** [review]
Just Add a new line at EOF and also please submit a Readme for this Script. You can have a look at this for your reference https://github.com/prathimacode-hub/IoT-Spot/blob/main/Arduino/Alternate%20LED%20Blinking/README.md

File: Atmel/LED Blinking/led_blinking.c (hunk 1 above)
Comment (khanasif786) [reply Atmel/LED Blinking/led_blinking.c:15]: @Harshil-Jani  in separate commit ?

Comment (Harshil-Jani) [reply Atmel/LED Blinking/led_blinking.c:15]: Anything that suits you. You can rebase and do the changes in same commit or maybe create a new commit and then squash all of them together.

Comment (khanasif786) [reply Atmel/LED Blinking/led_blinking.c]: @Harshil-Jani Done.

**Review by Harshil-Jani — APPROVED** [review]

**Pull request merged**